charts/helm-deployer/templates/clusterrole.yaml

```diff
@@ -0,0 +1,40 @@
+# SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+{{- if .Values.serviceAccount.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "deployer.fullname" . }}
+  labels:
+    {{- include "deployer.labels" . | nindent 4 }}
+rules:
+- apiGroups:
+  - landscaper.gardener.cloud
+  resources:
+  - deployitems
+  - deployitems/status
+  - targets
```

Comment:
the deployers do need update permissions for targets. read should be enough

Reply:
Fixed for all deployers, `targets` are now `get, list, watch` only.